apps/demo-react/pages/index.tsx

```diff
@@ -1,11 +1,13 @@
-import { Button } from '@lidofinance/lido-ui';
+import { uiReact } from 'reef-knot';
```

Comment:
can you use import { Button } from '@reef-knot/ui-react' ?

Reply:
Yes, thank you, forgot to update. Done